prospero-cli/src/main/resources/UsageMessages.properties

```diff
@@ -318,6 +321,8 @@ prospero.channels.added=Channel '%s' added.
 prospero.channels.remove.header=Unsubscribing %s from channel %s%n
 prospero.channels.removed=Channel '%s' removed.
 prospero.channels.error.notfound=The requested channel doesn't exist
+prospero.channels.error.morethanonechannel.found=More than one channel found
+prospero.channels.error.arguments.incorrect=Passing invalid arguments
```

Comment:
@parsharma the error message should let the user know what is the problem. In this case the problem is the command requires a channel definition (either via `--channel` or other parameters).